scripts/build-ancillary-repository.sh

```diff
@@ -61,6 +62,48 @@ function build_ancillary_repository() {
 	EOF
 }
 
+function compile_build_info() {
```

Comment:
How much of this build information is related to the ancillary repository? It seems awkward to have information unnecessary to the ancillary repository being emitted by this script.